src/edu/stanford/nlp/sempre/Parser.java

```diff
@@ -56,6 +56,9 @@
 
     @Option(gloss = "Dump all features (for debugging)")
     public boolean dumpAllFeatures = false;
+
+    @Option(gloss = "Whether rules are marked as floating by default")
+    public boolean defaultIsFloating = true;
```

Comment:
It seems better to have it in one place, instead of in both the Floating and ShiftReduceParser